x-pack/plugins/observability/public/hooks/use_get_filtered_rule_types.ts

```diff
@@ -6,13 +6,17 @@
  */
 
 import { useMemo } from 'react';
-import { ES_QUERY_ID } from '@kbn/rule-data-utils';
+import { ES_QUERY_ID, ML_ANOMALY_DETECTION_RULE_TYPE_ID } from '@kbn/rule-data-utils';
 import { usePluginContext } from './use_plugin_context';
 
 export function useGetFilteredRuleTypes() {
   const { observabilityRuleTypeRegistry } = usePluginContext();
 
   return useMemo(() => {
-    return [ES_QUERY_ID, ...observabilityRuleTypeRegistry.list()];
+    return [
+      ES_QUERY_ID,
+      ML_ANOMALY_DETECTION_RULE_TYPE_ID,
```

Comment:
@vinaychandrasekhar @katrin-freihofner Currently, the Anomaly rule is the second item in the rules list, since it is Beta, should we move it to the end of the list? (Also, in case of no ML job, the flyout does not have an option to send the user to create one.)
<img src="https://github.com/elastic/kibana/assets/12370520/c462c5d9-aa7e-4ab5-984a-dd7ee4518622" width=300 />

Reply:
I discussed this with Vinay, and the current order is good.